crates/edr_eth/src/remote/block_spec.rs

```diff
@@ -320,11 +320,11 @@ mod tests {
         help_test_block_spec_serde(BlockSpec::Tag(BlockTag::Pending));
         help_test_block_spec_serde(BlockSpec::Tag(BlockTag::Safe));
         help_test_block_spec_serde(BlockSpec::Eip1898(Eip1898BlockSpec::Hash {
-            block_hash: B256::from_low_u64_ne(1),
+            block_hash: B256::random(),
```

Comment:
What's the reason for changing to random here?

Reply:
There is no longer an API available for constructing from a `u64` directly, which means you first have to create a `U256` or `U160` (for `Address`) and then cast to the matching hash/byte type. I realised having a specific value here doesn't matter. We're merely testing the round trip, so any arbitrary value suffices. As such, I decided to use the `random` function.